docs/rfcs/1000-shamir-based-recovery.md

```diff
@@ -329,14 +329,11 @@ This secret contains both:
 
 ## 2 - Get the Shamir recovery setup
 
-This enables two use cases:
+The shamir related certificates are retrieved by the `get_certificates` route, with other certificates depending on the timestamp.
+If the user has authored a shamir setup, they will get the corresponding brief.
+If they are a share recipient, they will get their share and the associated brief.
 
-1) A User retrieving its own Shamir recovery setup. Useful for displaying it to
-   the user and to ensure the setup is still valid (i.e. no recipient has been
-   revoked)
-2) A user retrieving all the Shamir recovery information they are a part of.
-
-Authenticated API:
+> TODO: what happens with shamir deletion certificates.
```

Comment:
I don't think shamir deletion certificate are a special case here: they get provided just like brief and share ones (ordered by timestamp) and the client's certificates ops will take care of them (typically by checking a corresponding brief certificate exists before integrating them, which is already what is done for share certificate btw)